src/js/BookReader/Navbar/Navbar.js

```diff
@@ -0,0 +1,180 @@
+/** @typedef {import("../../BookReader.js").default} BookReader */
+
+export class Navbar {
+  /**
+   * @param {BookReader} br 
+   */
+  constructor(br) {
+    this.br = br;
+
+    /** @type {JQuery} */
+    this.$root = null;
+    /** @type {JQuery} */
+    this.$nav = null;
+    /** @type {number} */
+    this.maxPageNum = null;
+  }
+
+  /**
+   * Initialize the navigation bar (bottom)
+   * @param {Object} options
+   * @param {string} [options.navTitle]
+   * @return {JQuery}
+   */
+  init({ navTitle }) {
+    const { br } = this;
+
+    br.refs.$BRfooter = this.$root = $(`<div class="BRfooter"></div>`);
+    br.refs.$BRnav = this.$nav = $(
+      `<div class="BRnav BRnavDesktop">
+          <div class="BRnavCntl BRnavCntlBtm BRdn js-tooltip" title="Toggle toolbars"></div>
+          ${navTitle ? `<div class="BRnavTitle">${navTitle}</div>` : ''}
+          <div class="BRnavpos">
+            <div class="BRpager"></div>
+            <div class="BRnavline"></div>
+          </div>
+          <div class="BRpage">`
+
+        // Note, it's important for there to not be whitespace
+        + `<span class='BRcurrentpage'></span>`
+        + `<button class="BRicon book_left js-tooltip"></button>`
+        + `<button class="BRicon book_right js-tooltip"></button>`
+        + `<button class="BRicon onepg desktop-only js-tooltip"></button>`
+        + `<button class="BRicon twopg desktop-only js-tooltip"></button>`
+        + `<button class="BRicon thumb desktop-only js-tooltip"></button>`
+        // zoomx
+        + `<button class="BRicon zoom_out desktop-only js-tooltip"></button>`
+        + `<button class="BRicon zoom_in desktop-only js-tooltip"></button>`
+        + `<button class="BRicon full js-tooltip"></button>`
+        + `</div>
+        </div>`);
+  
+    this.$root.append(this.$nav);
+    br.refs.$br.append(this.$root);
+
+    const $slider = this.$root.find('.BRpager').slider({
+      animate: true,
+      min: 0,
+      max: br.getNumLeafs() - 1,
+      value: br.currentIndex(),
+      range: "min"
+    });
+
+    $slider.on('slide', (event, ui) => {
+      this.updateNavPageNum(ui.value);
+      return true;
+    });
+
+    $slider.on('slidechange', (event, ui) => {
```

Comment:
What do you think about coupling event handlers with the DOM string for BR in general - is this the way to go? or parse them out to another function?